### PR TITLE
Add pipeline

### PR DESCRIPTION
**Pull request by JrooTJunior**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  test-n-build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        python-version:
+          - 3.6
```
Comment (uwinx) [inline .github/workflows/build.yaml:21]: We need ^python3.6.1 (for pre-commit)

Comment (JrooTJunior) [reply .github/workflows/build.yaml:21]: Is not available from actions/setup-python - https://github.com/actions/python-versions/blob/master/versions-manifest.json#L332-L494

Comment (JrooTJunior) [reply .github/workflows/build.yaml:21]:
```
Run actions/setup-python@v2
Successfully setup CPython (3.6.10)
```

Comment (uwinx) [reply .github/workflows/build.yaml:21]: okay  thanks

**Pull request merged**